chart/values.yaml

```diff
@@ -42,6 +42,19 @@ operator:
   # Resource requests and limits for the operator pod
   # see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
   resources: {}
+  deployment:
+    # Custom labels for the Deployment resource.
+    customLabels:
+    # Custom labels for the Deployment Pod Template.
+    podTemplateCustomLabels:
+
+## Configuration for the tekton pruner cron job.
+pruner:
+  image:
+    # Container image for Tekton pruner. Defaults to gcr.
+    repository: ""
+    pullPolicy: IfNotPresent
```

Comment:
@cmorinupgrade does this pull policy used somewhere?

Reply:
Not it wasn't. I removed it. Thank you!